pkg/install/resources.go

```diff
@@ -105,8 +105,12 @@ func ServiceAccount(namespace string, annotations map[string]string) *corev1.Ser
 }
 
 func ClusterRoleBinding(namespace string) *rbacv1beta1.ClusterRoleBinding {
+	crbName := "velero"
+	if namespace != "velero" {
```

Comment:
Sorry I missed this before, but we should probably use the [`DefaultNamespace`](https://github.com/vmware-tanzu/velero/blob/main/pkg/apis/velero/v1/constants.go#L22) constant to compare against here. It looks like it might not be used throughout the rest of the file, but that's an oversight.

Reply:
Following the convention of Default vars, I added `DefaultVeleroNamespace` and used it in the unit tests. PTAL